core/templates/dev/head/pages/collection_editor/collection_editor.html

```diff
@@ -13,6 +13,7 @@
   <script type="text/javascript">
     GLOBALS.collectionId = JSON.parse('{{collection_id|js_string}}');
     GLOBALS.TAG_REGEX = JSON.parse('{{TAG_REGEX|js_string}}');
+    GLOBALS.INTERACTION_SPECS = JSON.parse('{{INTERACTION_SPECS|js_string}}');
```

Comment:
Why was this not needed before (but needed now)?

Reply:
This is needed for testing purposes. Previously, every file was loaded for tests, now only the files that are needed for testing the particular file, are loaded.